docker/caldera/README.md

```diff
@@ -0,0 +1,9 @@
+# Caldera Docker image
```

Comment:
It's less than ideal, but Caldera simply has no prebuilt image available. I think it's a bad idea to push to the Docker Hub ourselves, so we're stuck with the user having to build locally. We make it as easy as possible with a `make` target.

Reply:
You can build a docker image from a remote repository by providing the url to the build in a docker compose. See [this SO](https://stackoverflow.com/questions/66448551/docker-compose-build-image-from-git-repo).

Reply:
I had something like this while I was testing docker:
```yaml
  caldera:
    container_name: caldera_server
    build: https://github.com/mitre/caldera.git
    ports:
      - 8443:8443
      - 8888:8888
```